lib/versioned/test.js

```diff
@@ -101,9 +101,34 @@ Test.prototype.peek = function peek() {
 }
 
 Test.prototype.run = function run() {
+  const reportMissing = () => {
+    if (this.missingFiles.length) {
+      // warn or throw if files which exist are not specified.
+      const plural = this.missingFiles.length === 1 ? '' : 's'
+      /* eslint-disable no-console */
+      console.log(
```

Comment:
this is showing up on every iteration and not just at end
```sh
===============================================================
 • express/route-iteration.tap.js run 12 of 16): running
===============================================================
Found 1 test file outside of those specified:

===============================================================
 • express/route-param.tap.js run 13 of 16): running
===============================================================
Found 1 test file outside of those specified:

===============================================================
 • express/router-params.tap.js run 14 of 16): running
===============================================================
Found 1 test file outside of those specified:

===============================================================
 • express/segments.tap.js run 15 of 16): running
===============================================================
Found 1 test file outside of those specified:

===============================================================
 • express/transaction-naming.tap.js run 16 of 16): running
===============================================================
Found 1 test file outside of those specified:

- /Users/revans/code/node-newrelic/test/versioned/express/unused.tap.js

Include them in a test group's package.json or delete them.
Found 1 test file outside of those specified:
```

Reply:
it works fine if you pass in `--strict`

```sh
===============================================================
 • express/app-use.tap.js run 1 of 16): running
===============================================================
Found 1 test file outside of those specified:

- /Users/revans/code/node-newrelic/test/versioned/express/unused.tap.js

Include them in a test group's package.json or delete them.
```

Reply:
This module is very confusing because it's just updating stdout which is done in the printers.  I think that's where you're going to need to put the loggic on printing out the message if there are missing tests and if in strict failing the test run